mirror-godot-app/ui/main_menu/common/space_panel.gd

```diff
@@ -7,8 +7,8 @@ signal space_pressed(space: Dictionary)
 @onready var _preview_image: UrlTextureRect = %Preview
 @onready var _title_label: Label = %TitleLabel
 @onready var _description_label: Label = %DescriptionLabel
-@onready var _is_ready_zone: Control = %IsReadyZone
-@onready var _present_users_in_build = %PresentUsersInBuild
+#@onready var _is_ready_zone: Control = %IsReadyZone
```

Comment:
ask for liza to test the panels and check they are correct and working

AFAIK these values always error when used even when uncommented and this doesn't work but we might want to remove this change.